collab-entity/src/proto/collab.rs

```diff
@@ -1,4 +1,3 @@
-// This file is @generated by prost-build.
```

Comment:
I guess this was unintentional. For some reason this change also happens to me 🥲